src/components/NodeEditable.tsx

```diff
@@ -110,12 +139,15 @@ class NodeEditable extends Component<Props> {
     }
   };
 
-  suppressEvent = (e: React.SyntheticEvent) => {
-    e.stopPropagation();
-  };
-
   componentDidMount() {
-    this.setSelection(this.props.isInsertion);
+    this.pendingTimeout = setAfterDOMUpdate(() => {
+      const element = this.element.current;
+      if (!element) {
+        // element has been unmounted already, nothing to do.
+        return;
+      }
+      selectElement(element, this.props.isInsertion);
+    });
```

Comment:
I'm not sure why this use of `setAfterDOMUpdate` is necessary, but the tests failed when I tried to remove it. So something else is implicitly depending on `NodeEditable` having this asynchronous side effect upon mount. All the logic around focusing seems really quite complicated.